mono/utils/mono-cgroup.c

```diff
@@ -0,0 +1,958 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+/*++
+
+Module Name:
+
+    mono-cgroup.cpp
+
+Abstract:
+    Read the memory limit for the current process
```

Comment:
would be nice to mention that this was ported from coreclr and mention the exact commit that was used (to make it easier to port additional commits).

Reply:
Added.